### PR TITLE
Updates to query-types section

### PR DESCRIPTION
**Pull request by abhinavdangeti**

### CODE REVIEW

**Comment by abhinavdangeti** [comment]
Ping @amarantha-k for review.

File: modules/fts/pages/fts-query-types.adoc [hunk 1]
```diff
@@ -62,7 +62,8 @@ For more information on using the REST API to perform queries, see xref:fts-sear
 
 A match query _analyzes_ input text, and uses the results to query an index.
 Options include specifying an analyzer, performing a _fuzzy match_, and performing a _prefix match_.
-By default, the analyzer used for the input-text is that previously used on the target-text, during index-creation.
+By default, the analyzer used for the search text is what was set for the specified field, during index-creation.
```
Comment (amarantha-k) [inline modules/fts/pages/fts-query-types.adoc]: during index creation. (no hyphen)

Comment (amarantha-k) [reply modules/fts/pages/fts-query-types.adoc]: Just noticed that FTS docs use "index-creation". Unless this is a FTS-specific usage, we should update to "index creation"

**Review by amarantha-k — CHANGES_REQUESTED** [review]

File: modules/fts/pages/fts-query-types.adoc (hunk 1 above)
Comment (abhinavdangeti) [reply modules/fts/pages/fts-query-types.adoc]: Done.

**Review by amarantha-k — APPROVED** [review]
lgtm

**Pull request merged**